packages/babel-plugin-proposal-logical-assignment-operators/src/index.js

```diff
@@ -39,13 +39,26 @@ export default declare(api => {
           }
         }
 
-        path.replaceWith(
-          t.logicalExpression(
-            operator.slice(0, -1),
-            lhs,
-            t.assignmentExpression("=", left, right),
-          ),
-        );
+        const isRHSAnonymousFunction = t.isFunctionExpression(right, {
+          id: null,
+        });
+        if (isRHSAnonymousFunction && operator === "??=") {
```

Comment:
We need to apply this to all the operators, not just `??=`.

Reply:
Ah, yeah.